spring-cloud-stream-reactive/src/test/java/org/springframework/cloud/stream/reactive/StreamListenerTestGenericFluxInputOutputArgsWithMessage.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.reactive;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import reactor.core.publisher.Flux;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.cloud.stream.annotation.EnableBinding;
+import org.springframework.cloud.stream.annotation.Input;
+import org.springframework.cloud.stream.annotation.Output;
+import org.springframework.cloud.stream.annotation.StreamListener;
+import org.springframework.cloud.stream.messaging.Processor;
+import org.springframework.cloud.stream.test.binder.MessageCollector;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.messaging.support.MessageBuilder;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+@RunWith(Parameterized.class)
+@SuppressWarnings("unchecked")
+public class StreamListenerTestGenericFluxInputOutputArgsWithMessage {
+
+	private Class<?> configClass;
+
+	public StreamListenerTestGenericFluxInputOutputArgsWithMessage(Class<?> configClass) {
+		this.configClass = configClass;
+	}
+
+	@Parameterized.Parameters
+	public static Collection InputConfigs() {
+		return Arrays.asList(new Class[] {TestGenericStringFluxInputOutputArgsWithMessageImpl1.class});
```

Comment:
will remove parameterization here. this was the result of previous commit that used multiple supported options.